source/Missions/BoardGameLogic.cs

```diff
@@ -103,40 +106,30 @@ private void PreplaceUnits()
             MovePawnToTileDelayedMethod.Invoke(seegaBoardGame, new object[] { seegaBoardGame.PlayerOneUnits[1], seegaBoardGame.GetTile(2, 4), false, false, 1f });
         }
 
-        private void SeegaPawnCapture(PawnBase pawn)
+        private void OnPawnCapture(PawnBase pawn)
         {
             //Only call SetPawnCaptured when it's a forceful remove as a result of no moves available as otherwise it gets handled locally from the move
-            if (!FocusBlockingPawnsPatch.ForceRemove)
-            {
-                return;
-            }
+            //if (!FocusBlockingPawnsPatch.ForceRemove)
```

Comment:
nit: commented out code should be deleted?

Reply:
Want to keep these comments as I will need them almost certainly when fixing the other board games, particularly Seega